src/services/pcn-packetcapture/src/Packetcapture.cpp

```diff
@@ -105,7 +116,17 @@ void Packetcapture::writeDump(const std::vector<uint8_t> &packet){
     myFile.write(reinterpret_cast<const char*>(pcap_header), sizeof(*pcap_header));
     writeHeader = false;
   }
-  gettimeofday(&tp, NULL);
+  struct timeval tp;
```

Comment:
Add a comment such as "The algorithm for timestamping packets is described in file XX function YY"